src/components/infinite-data-table.jsx

```diff
@@ -0,0 +1,427 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  CellMeasurer,
+  CellMeasurerCache,
+  MultiGrid,
+  AutoSizer,
+  InfiniteLoader,
+} from 'react-virtualized';
+import { serializableDeepAreEqual } from '../utils';
+import '../styles/components/data-table.scss';
+
+const DEFAULT_WIDTH = 200;
```

Comment:
Are these table width/height or cell? Please add to variable name

Reply:
Done.